.github/workflows/export-repo-secrets.yml

```diff
@@ -0,0 +1,25 @@
+permissions: write-all # Equivalent to default permissions plus id-token: write
+name: Export secrets to ESC
+on: [ workflow_dispatch ]
```

Comment:
If you just do 
```
on:
  pull_request:
```
wouldn't this just run in the PR so you don't need to merge it at all?